ImagePickerSheetController/Example/ViewController.swift

```diff
@@ -51,6 +51,7 @@ class ViewController: UIViewController, UIImagePickerControllerDelegate, UINavig
             }
             
             let controller = ImagePickerSheetController()
+            controller.limit = 1
```

Comment:
I suggest we don't put this in the example app to make it look like the iMessage app.